apps/src/studio/studio.js

```diff
@@ -1894,43 +1884,56 @@ Studio.reset = function(first) {
   for (i = 0; i < Studio.spriteCount; i++) {
     Studio.displaySprite(i);
   }
+  this.resetGoalSprites();
```

Comment:
Diff gets weird here as I extracted a large chunk of `reset()` into `resetGoalSprites()`.